.gitignore

```diff
@@ -121,4 +121,5 @@ venv.bak/
 
 # Front
 node_modules
+marsha/static/js
 data/static/js
```

Comment:
isn't data/static/js the same purpose?

Reply:
This is the issue we had at the beginning of Richie: content in the data/static folder is not served during development, but content in {app}/static is

Reply:
IC